website/docs/r/edgegateway.html.markdown

```diff
@@ -58,13 +114,23 @@ The following arguments are supported:
 * `org` - (Optional) The name of organization to which the VDC belongs. Optional if defined at provider level.
 * `vdc` - (Optional) The name of VDC that owns the edge gateway. Optional if defined at provider level. 
 * `name` - (Required) A unique name for the edge gateway.
-* `external_networks` - (Required) An array of external network names.
+* `external_networks` - (Deprecated, Optional) An array of external network names. This supports
+  simple external networks with one subnet only. 
+* `external_network` - (Optional, *v2.6+*) One or more blocks defining external networks, their
+  subnets, IP addresses and  IP pool suballocation attached to edge gateway interfaces. Details are
+  in [external network](#external-network) block below.
 * `configuration` - (Required) Configuration of the vShield edge VM for this gateway. One of: `compact`, `full` ("Large"), `x-large`, `full4` ("Quad Large").
-* `default_gateway_network` - (Optional) Name of the external network to be used as default gateway. It must be included in the
+* `default_gateway_network` - (Deprecated, Optional) Name of the external network to be used as default gateway. It must be included in the
```

Comment:
Which field replaces this deprecated one?
In general, it feels we should add a sentence to all deprecated fields saying "Please use ... instead" to avoid putting the user into detective clothes.

Reply:
Added pointers - please check if it makes sense to have them at the end of description or I should move the new field descriptions to the front.